docs/conf.py

```diff
@@ -61,7 +61,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'F5 OpenStack LBaaSv2 Driver'
+project = u'F5 OpenStack LBaaSv2 Plugin'
```

Comment:
Deferring to @swormke on this one. I used the word plugin because that's what we used for LBaaSv1, but I'm open to suggestions. The point of the change is that the docs generated from this repo don't only apply to the driver itself, they're for our overall lbaasv2 solution.